src/com/facebook/buck/swift/SwiftCompile.java

```diff
@@ -471,4 +474,9 @@ public SourcePath getObjCGeneratedHeaderPath() {
   public SourcePath getOutputPath() {
     return ExplicitBuildTargetSourcePath.of(getBuildTarget(), outputPath);
   }
+
+  public SourcePath getSwiftModuleOutputPath() {
```

Comment:
can we add a `/***/` comment here to explain when this method should be used? Similar to the one above?

Reply:
👍